core/src/test/java/io/grpc/internal/ClientCallImplTest.java

```diff
@@ -120,6 +124,109 @@ public void advertisedEncodingsAreSent() {
     assertEquals(decompressorRegistry.getAdvertisedMessageEncodings(), acceptedEncodings);
   }
 
+  @Test
+  public void prepareHeaders_authorityAdded() {
```

Comment:
Yeah, it's a thing, specifically for separating the method name from the description of the test.